lib/middleware/blp-session.ts

```diff
@@ -20,6 +20,7 @@ function createSession(sessionId: string): Promise<blpapi.Session>
 {
     if (!(sessionId in SESSION_STORE)) {
         var s = new blpapi.Session(DEFAULT_SESSION_OPTIONS);
+        s.setMaxListeners(0);   // Remove max listener limit for blpSession
```

Comment:
Let's move to using `.defaultMaxListeners` in `blpapi-wrapper.ts`, which can be a 'class wide' property that is the default for all new EventEmitter instances.